salt/roster/clustershell.py

```diff
@@ -40,13 +34,13 @@ def targets(tgt, tgt_type="glob", **kwargs):
     ports = __opts__["ssh_scan_ports"]
     if not isinstance(ports, list):
         # Comma-separate list of integers
-        ports = list(map(int, six.text_type(ports).split(",")))
+        ports = list(map(int, str(ports).split(",")))
```

Comment:
**suggestion** probably not worth holding up this PR, but this looks an awful lot like it should be `ports = [int(port) for port in str(ports).split(",')]`